### PR TITLE
Slice days

### PR DESCRIPTION
**Pull request by yoshihara**
It takes long time to fetch data between `from_date` to `to_date` by one request. So I implemented this plugin fetches data between each 7 days.

### CODE REVIEW

File: lib/embulk/input/mixpanel.rb [hunk 1]
```diff
@@ -67,26 +81,40 @@ def init
...
+          to_date = dates.last
+          Embulk.logger.info "Fetching data from #{from_date.to_s} to #{to_date.to_s} ..."
+
+          params = @params.dup
+          params["from_date"] = from_date.to_s
+          params["to_date"] = to_date.to_s
+
+          records = client.export(params)
+
+          records.each.with_index do |record, i|
```
Comment (uu59) [inline lib/embulk/input/mixpanel.rb]: `with_index` isn't needed?

File: test/embulk/input/test_mixpanel.rb [hunk 2]
```diff
@@ -5,15 +5,53 @@
 module Embulk
   module Input
     class MixpanelTest < Test::Unit::TestCase
+      API_KEY = "api_key".freeze
+      API_SECRET = "api_secret".freeze
+      FROM_DATE = "2015-02-22".freeze
+      TO_DATE = "2015-03-02".freeze
+
+      DURATIONS = [
+        {from_date: FROM_DATE, to_date: "2015-02-28"}, # It has 7 days between 2015-02-02 and 2015-02-28
```
Comment (uu59) [inline test/embulk/input/test_mixpanel.rb]: s/02-02/02-22/ ?

Comment (yoshihara) [inline test/embulk/input/test_mixpanel.rb]: Oops :dizzy_face:

**Comment by uu59** [comment]
LGTM

**Pull request merged**